### PR TITLE
Store morden db and keys in "path/to/parity/data/Morden" (ropsten uses "test", like before)

### LINKED ISSUE #6159: Morden and Ropsten share db- and keys-path
Morden and Ropsten share db- and keys-path. DB is no issue, but ropsten keys and morden keys are both placed into a "test" directory.

### PR DESCRIPTION
**Pull request by dvdplm**
Fix #6159

### CODE REVIEW

File: ethcore/res/ethereum/morden.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
 	"name": "Morden",
-	"dataDir": "test",
+	"dataDir": "Morden",
```
Comment (sorpaas) [inline ethcore/res/ethereum/morden.json]: Can we change this to lower-case name `morden` to keep it consistent with other chain configs?

**Comment by sorpaas** [comment]
This should be marked releasenotes because it will require existing morden users to migrate their folders.

File: ethcore/res/ethereum/morden.json (hunk 1 above)
Comment (niklasad1) [reply ethcore/res/ethereum/morden.json]: @sorpaas you are referring to `"dataDir"` right? If, that's the case I totally agree:

```
classic.json:3: "dataDir": "classic",
easthub.json:3:  "dataDir": "easthub",
ellaism.json:3:  "dataDir": "ellaism",
expanse.json:3: "dataDir": "expanse",
foundation.json:3:      "dataDir": "ethereum",
kovan.json:3:   "dataDir": "kovan",
kovan_wasm_test.json:3: "dataDir": "kovan-test",
mcip3_test.json:3:  "dataDir":"mcip3test",
mcip6_byz.json:3:  "dataDir":"mcip6test",
morden.json:3:  "dataDir": "test",
musicoin.json:3:  "dataDir":"musicoin",
ropsten.json:3: "dataDir": "test",
social.json:3:  "dataDir": "social",
```

Comment (dvdplm) [reply ethcore/res/ethereum/morden.json]: Changed. :)

**Pull request merged**